src/home_robot/home_robot/core/interfaces.py

```diff
@@ -55,6 +55,7 @@ class Observations:
     semantic: Optional[
         np.array
     ] = None  # (camera_height, camera_width) in [0, num_sem_categories - 1]
+    camera_extrinsic: Optional[np.array] = None  # (4, 4) camera pose in world coordinates
```

Comment:
Maybe just `camera_pose`? So it would be coherent with other fields that might be added to the Observation (`base_pose`, `ee_pose`, etc.)